config/filter.d/apache-common.conf

```diff
@@ -22,6 +22,9 @@ apache-prefix-syslog = %(__prefix_line)s
 apache-prefix-logfile = \[\]\s
 apache-prefix-all = (?:%(apache-prefix-logfile)s|%(apache-prefix-syslog)s)?
 
+# Setting for __prefix_line (only `logging=syslog`):
+_daemon = (?:apache\d*|httpd(?:/\w+)?)
```

Comment:
Apache now has an option to configure the syslog tag :
https://httpd.apache.org/docs/trunk/mod/core.html#errorlog
It may then be something different than `apache|httpd`.